src/models/stores/user.ts

```diff
@@ -36,7 +36,15 @@ export const UserModel = types
     className: "",
     classHash: "",
     offeringId: "",
+    // This is the last group the user joined. It is synced with the latestGroupId
+    // property in Firebase
     latestGroupId: types.maybe(types.string),
+    // This is the group of this user in the particular offering that is being
+    // run right now. There is a view which will return this value if set otherwise
+    // it returns latestGroupId. There might be a period when CLUE is being loaded
```

Comment:
>There is a view which will return this value if set otherwise it returns latestGroupId.

I'm not seeing such a view, but it seems like it could be useful.

Reply:
Oops. I dropped that approach. It seemed better to fail fast and let code using it decide how to handle the case when the group wasn't known yet.